src/components/Table/GridTable.tsx

```diff
@@ -905,16 +912,11 @@ export interface RowStyle<R extends Kinded> {
 function getIndentationCss<R extends Kinded>(
   style: GridStyle,
   rowStyle: RowStyle<R> | undefined,
-  columnIndex: number,
   maybeContent: ReactNode | GridCellContent,
+  applyRowIndent: boolean,
 ): Properties {
-  // Look for cell-specific indent or row-specific indent (row-specific is only one the first column)
-  const indent = (isGridCellContent(maybeContent) && maybeContent.indent) || (columnIndex === 0 && rowStyle?.indent);
-  if (typeof indent === "number" && style.levels !== undefined) {
-    throw new Error(
-      "The indent param is deprecated for new beam fixed & flexible styles, use beamNestedFixedStyle or beamNestedFlexibleStyle",
```

Comment:
Hm, I dunno, maybe we could stay with having this defined via `style.levels`?

...I think the idea is that we have three ways of specifying behavior:

* `GridCellContent` --> very cell specific
* `RowStyle` --> based on the kind or data within the row
* `Style.levels` --> "this level is always ..."

And it kinda seems like `Style.levels` would be most what we want here, i.e.:

```
export const beamNestedFixedStyle: GridStyle = {
  ...beamFixedStyle,
  levels: { 3: { indentCss: Css.pl3 } },
};
```

So that a page doesn't have to define their own `RowStyle`s b/c the behavior really isn't based on rows/content.

I guess maybe we want to support all three, for more complex pages like Schedules V2 that are going to intermix regular rows with special behavior rows...

Reply:
Hmm. I get what you're saying. Yea, schedules V2 has different row types at the same `level`, which could hinder this sort of thing, though for that they can always just use rowStyles as is done today to get around it, even without the `indent` property. 
I'm okay with updating this to be something more like your example. Though, rather than calling it `indentCss`, I wonder if it'd be something like, `firstNonActionColumn` and we'd apply both padding & type scale,  `Css.pl3.tiny.$. What do you think?

Reply:
Yeah agreed it'd be great to do `Css.pl3.tiny.$`... Maybe `firstContentColumn`?

Reply:
Updated to follow the suggestion. I think it's much cleaner. Thank you!